src/components/button/component.js

```diff
@@ -430,6 +431,18 @@ export let Button = xcomponent.create({
                 if (style.label === 'buynow' && style.size === 'tiny') {
                     throw new Error(`Invalid ${style.label} button size: ${style.size}`);
                 }
+
+                if (style.label === 'buynow' && style.hasOwnProperty('fundingicons') && style.hasOwnProperty('branding')) {
```

Comment:
I wanna chat internally about this -- my feeling is we should allow funding icons for any kind of button (except credit)